packages/cli/internal/pkg/cli/workflow/manager.go

```
@@ -436,6 +475,31 @@ func (m *Manager) setWorkflowParameters() {
 	m.workflowParams["workflowInputs"] = filepath.Base(m.attachments[0])
 }
 
+func (m *Manager) setWorkflowEngineParameters() {
+	if m.err != nil {
+		return
+	}
+	m.workflowEngineParams = make(map[string]string)
+	if m.optionFileUrl == "" {
+		return
+	}
+	if m.options != nil {
+		if m.workflowEngine == "nextflow" || m.workflowEngine == "miniwdl" {
+			log.Error().Msgf("optionFile flag cannot be used with head node engines")
```

Please return an error (`m.err = ...`) rather than log one and continue